assets/js/Containers/FrevaGPT/index.js

```diff
@@ -0,0 +1,438 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+
+import {
+  Container,
+  Row,
+  Col,
+  FormControl,
+  InputGroup,
+  Button,
+  Form,
+  Alert,
+  Tooltip,
+  OverlayTrigger,
+} from "react-bootstrap";
+
+import { browserHistory } from "react-router";
+import { isEmpty, has } from "lodash";
```

Comment:
I think, that `has` is not necessary as JavaScript has its own built-in implementation by now: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn